### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by bhopmann**

### CODE REVIEW

File: Calendr/Assets/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -86,10 +86,10 @@
 "reminder.options.complete" = "Erledigt";
 "reminder.options.remind" = "Erinnerung %@";
 
-"event_details.participant.organizer" = "Veranstalter";
-"event_details.participant.me" = "mich";
+"event_details.participant.organizer" = "Veranstaltende Person";
+"event_details.participant.me" = "Ich";
 
-"event_status.label" = "mein status:";
+"event_status.label" = "Mein status:";
```
Comment (pakerwreah) [inline Calendr/Assets/de.lproj/Localizable.strings:92]: This one was lowercase on purpose, to align with the native Calendar UI.

**Review by pakerwreah — CHANGES_REQUESTED** [review]

File: Calendr/Assets/de.lproj/Localizable.strings (hunk 1 above)
Comment (bhopmann) [reply Calendr/Assets/de.lproj/Localizable.strings:92]: Apple Calendar is starting with capital letter as well:

Bildschirmfoto 2024-11-11 um 16 47 03

Comment (pakerwreah) [reply Calendr/Assets/de.lproj/Localizable.strings:92]: geez, german is confusing 🤣

**Comment by pakerwreah** [comment]
There's a new one:
`//"calendars.source.others" = "Others";`

Could you translate it as well?

**Review by pakerwreah — APPROVED** [review]

**Comment by pakerwreah** [comment]
Don't worry, I just did it. 🙇🏻‍♂️

**Pull request merged**